packages/backend-modules/assets/express/pdf.js

```diff
@@ -17,15 +17,17 @@ module.exports = (server) => {
 
     // pick query parameters intended for PDF endpoint
     // (pass along rest as options, later)
-    const { images, download, size, ...options } = req.query
+    const { images, download, format, ...options } = req.query
 
     // build URL to fetch PDF from
+    // TODO: PDF_BASE_URL = https://screenshot.republik.ch/api/pdf
     const fetchUrl = new URL(`${PDF_BASE_URL}/${path}`)
```

Comment:
I _think_ this should probably just be `${PDF_BASE_URL}`, since the path is set on the `url` param?